README.md

```diff
@@ -11,86 +11,136 @@
 ```sql
 {{ salesforce_formula_utils.sfdc_formula_view(
     source_table='your_source_table_name_here', 
     fields_to_include=['i_want_this_field','also_this_one','maybe_a_third_as_well','lets_add_more'],
     full_statement_version=false) 
 }}
 ```
-### Formula Fields that Reference Other Formula Fields
-This macro has been created to allow for two degrees of formula field reference. For example:
+
+**Output**: Only formulas provided in the `fields_to_include` variable will be included in the resulting `select` statement for the chosen source table.
+
+> This option makes use of the `fivetran_formula` lookup table, which requires the package to combine fields' formulas into a SQL query for each source table. This option does not support double-nested formulas and therefore may be incomplete compared to Option #1.
+
+### Automate model creation
+If you have multiple models you need to create, you can also leverage the [sfdc_formula_model_automation](https://github.com/fivetran/dbt_salesforce_formula_utils#sfdc_formula_model_automationsh-source) script within this project to automatically create models locally via the command line. Below is an example command to copy and edit.
+
+```bash
+source dbt_modules/salesforce_formula_utils/sfdc_formula_model_automation.sh "../path/to/directory" "desired_table_1,desired_table_2,desired_table_infinity"
+```
+
+**Output**: Model files for each table, populated with `{{ salesforce_formula_utils.sfdc_formula_view(source_table='table_name') }}`. By default, these models will run Option #1. To use Options #2 or #3, you will need to add `full_statement_version=false`. For Option #3, you will need to add the `fields_to_include` argument(s).
+
+> Note: In order for this command to work, you must currently be within the root directory of your dbt project. 
+
+## Step 5: Exclude problematic formula fields
+The `sfdc_formula_view` macro has been created to support double-nested formula field references for most cases using Option #1, however, never if using Options #2 or #3. For example:
 - :white_check_mark: : A formula field references standard fields from the base Salesforce table.
-- :white_check_mark: : A formula field references another formula field that does not reference other formula fields.
-- 🚧     : A formula field references another formula field that references another formula field (etc.). This may be possible for certain situations using Option #1 above.
+- :white_check_mark: : A formula field references another formula field that does **not** reference other formula fields.
+The `sfdc_formula_view` macro may be able to support some cases of n-nested formula field references if using Option #1 above, however, never if using Options #2 or #3. For example:
```

Comment:
Think you just need a white space before this line or else the spacing turns out a bit weird in the final readme output
```suggestion

The `sfdc_formula_view` macro may be able to support some cases of n-nested formula field references if using Option #1 above, however, never if using Options #2 or #3. For example:
```